lms/templates/base.html

```diff
@@ -9,6 +9,18 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='my.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='prism.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='hawcons.css') }}">
+    <link rel="manifest" href="{{ url_for('static', filename='manifest.json') }}">
+    <meta name="apple-mobile-web-app-capable" content="yes">
```

Comment:
This is lots of code for just a fallback, I think we should shrink it to be as slim as the W3C example [here](https://www.w3.org/TR/appmanifest/#using-a-link-element-to-link-to-a-manifest).